inc/saihostif.h

```diff
@@ -454,6 +454,14 @@ typedef enum _sai_hostif_trap_attr_t
      */
     SAI_HOSTIF_TRAP_ATTR_MIRROR_SESSION,
 
+    /**
+     * @brief Number of packets received for the trap
+     *
+     * @type sai_uint64_t
+     * @flags READ_ONLY
+     */
+    SAI_HOSTIF_TRAP_ATTR_RX_PKTS,
```

Comment:
since trap can be set to drop, forward, does this count all cases or it only counts packets with action trap?

Reply:
The intent was to count only the packets trapped to CPU. If there is a need, I am open to generalise it to count all packets that hit the trap.